x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/action_connectors/import_rule_action_connectors.test.ts

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { actionsClientMock } from '@kbn/actions-plugin/server/actions_client.mock';
+import {
+  getImportRulesSchemaMock,
+  webHookConnector,
+} from '../../../../../../../common/detection_engine/rule_management/model/import/rule_to_import.mock';
+import { importRuleActionConnectors } from './import_rule_action_connectors';
+import { coreMock } from '@kbn/core/server/mocks';
+
+const rules = [
+  {
+    ...getImportRulesSchemaMock(),
+    actions: [
+      {
+        group: 'default',
+        id: 'cabc78e0-9031-11ed-b076-53cc4d57aaf1',
+        action_type_id: '.webhook',
+        params: {},
+      },
+    ],
+  },
+];
+const rulesWithoutActions = [
+  {
+    ...getImportRulesSchemaMock(),
+    actions: [],
+  },
+];
+const actionConnectors = [webHookConnector];
+const actionsClient = actionsClientMock.create();
+actionsClient.getAll.mockResolvedValue([]);
+const core = coreMock.createRequestHandlerContext();
+
+describe('checkRuleExceptionReferences', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('should show an error message when the user has a Read Actions permission and stops the importing ', async () => {
+    // jest.resetAllMocks();
```

Comment:
Should this still be commented out?

P.S. thank you for all the test coverage here 🙂

Reply:
Good catch it should be removed actually because I am reusing the mocked module in all tests but change the content based on the test scenario